topics/visualisation/faqs/visualizations_jbrowse.html

```diff
@@ -1,3 +1,7 @@
+---
```

Comment:
Title? Description? Layout?

Reply:
I don't think this should be a faq, this is a snippet, I think we should move this back. it's not like the other faqs, it's very special to jbrowse tutorials (and shouldn't be in viz, it's used in genome annotation, sequence analysis as well.)

Reply:
currently it won't show up in the FAQ page because we don't set the `layout` property here (that's why I left it out), but we can move those to a special folder too if we want yeah